plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -237,7 +239,46 @@ public static void main(String[] args)
                     .build();
 
             Thread.sleep(10);
-            Logger log = Logger.get(IcebergMinIoHiveMetastoreQueryRunnerMain.class);
+            Logger log = Logger.get(IcebergMinioHiveMetastoreQueryRunnerMain.class);
+            log.info("======== SERVER STARTED ========");
+            log.info("\n====\n%s\n====", queryRunner.getCoordinator().getBaseUrl());
+        }
+    }
+
+    public static final class IcebergMinioQueryRunnerMain
+    {
+        private IcebergMinioQueryRunnerMain() {}
+
+        public static void main(String[] args)
+                throws Exception
+        {
+            Logging.initialize();
+
+            String bucketName = "test-bucket";
+            Minio minio = Minio.builder().build();
+            minio.start();
+            minio.createBucket(bucketName);
+
+            DistributedQueryRunner queryRunner = IcebergQueryRunner.builder()
+                    .setCoordinatorProperties(Map.of(
+                            "http-server.http.port", "8080"))
+                    .setIcebergProperties(Map.of(
+                            "iceberg.catalog.type", "TESTING_FILE_METASTORE",
+                            "hive.metastore.catalog.dir", "s3://%s/".formatted(bucketName),
+                            "hive.s3.aws-access-key", MINIO_ACCESS_KEY,
+                            "hive.s3.aws-secret-key", MINIO_SECRET_KEY,
+                            "hive.s3.endpoint", "http://" + minio.getMinioApiEndpoint(),
+                            "hive.s3.path-style-access", "true",
+                            "hive.s3.streaming.part-size", "5MB"))
+                    .setSchemaInitializer(
+                            SchemaInitializer.builder()
+                                    .withSchemaName("tpch")
+                                    .withClonedTpchTables(TpchTable.getTables())
+                                    .build())
+                    .build();
+
+            Thread.sleep(10);
```

Comment:
I believe the sleep is redundant (at least ATQF won't sleep after creating a query runner, so it works without a sleep in tests).

will remove it in https://github.com/trinodb/trino/pull/16714, where i was adding similar runner